R/utils.R

```diff
@@ -103,8 +104,8 @@ get_content <- function(lines, info) {
   paste(lines, collapse = "\n")
 }
 
-logical_env <- function(x) {
-  res <- as.logical(Sys.getenv(x))
+logical_env <- function(x, unset = "") {
```

Comment:
Nocov this function or is it used after namespace loading?

Reply:
Yea it's only in `.onLoad()`:

https://github.com/r-lib/lintr/blob/e01791f4e02195fc6ab39b9ff785de5a288b74db/R/zzz.R#L322

Reply:
Moved it next to `.onLoad()` for clarity, also gets `#nocov`'d.